### DISCUSSION TITLE
Finding Beginner-Friendly Projects to Contribute to

### DISCUSSION THREAD

**Discussion by JoshuyPan** [Cyfrin/foundry-full-course-cu#2107] [General]
Hey guys,

How can i find beginner-friendly projects that i can contribute to?

**Comment by EngrPips** [comment]
Start with contributing to the tools you use  `foundry,` `openzeppelin-contracts` and any other tool that has been useful for your development journey

**Reply by JoshuyPan** [reply]
thanks buddy

**Reply by EngrPips** [reply]
You are welcome, Friend.
